regulations/static/regulations/js/source/resources.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
It's not super clear to me on first blush what a resources file does. Can we add some descriptive comments? Will this only be for this versionElements object? If so, should we rename it `version-resources' or something similar?